app/src/main/java/com/github/se/icebreakrr/ui/profile/ProfileEdit.kt

```diff
@@ -56,7 +57,8 @@ import com.google.firebase.ktx.Firebase
 fun ProfileEditingScreen(
     navigationActions: NavigationActions,
     tagsViewModel: TagsViewModel,
-    profilesViewModel: ProfilesViewModel
+    profilesViewModel: ProfilesViewModel,
+    auth: FirebaseAuth = FirebaseAuth.getInstance()
```

Comment:
`auth: FirebaseAuth = FirebaseAuth.getInstance()`
Question: auth is not used in this screen, why add it in its signature?